app/renderer/components/main/loginRequired.js

```diff
@@ -133,26 +133,23 @@ class LoginRequired extends React.Component {
                   />
                 </div>
               </div>
-              : null
             }
           </div>
         </CommonFormSection>
         <CommonFormButtonWrapper>
           <Button l10nId='cancel' className='whiteButton' onClick={this.onClose} />
-          <Button l10nId='ok' className='primaryButton' onClick={this.onSave.bind(this)} />
+          <Button l10nId='ok' className='primaryButton' onClick={this.onSave} />
         </CommonFormButtonWrapper>
       </CommonForm>
     </Dialog>
   }
 }
 
-LoginRequired.propTypes = { frameProps: PropTypes.object }
+module.exports = LoginRequired
```

Comment:
@NejcZdovc please document the rule on the wiki page for refactoring work, thanks. CC @cezaraugusto